test/integration/rules/duplicate-id-active/duplicate-id-active.html

```diff
@@ -0,0 +1,19 @@
+<p id="ignore1">This is my first paragraph with this ID.</p>
+<div style="display:none">
+  <p id="ignore1">This is my second paragraph with this ID.</p>
+</div>
+
+<button id="fail1" class="fail1"></button>
+<button id="fail1"></button>
```

Comment:
Should we have more examples of failures? For example a `<div tabindex="-1" id="fail2">blah</div><input id="fail2" />` etc.?